lib/restforce/sobject.rb

```diff
@@ -22,11 +22,13 @@ def describe_layouts(layout_id = nil)
     #   account.Name = 'Foobar'
     #   account.save
     def save
+      ensure_write_access
```

Comment:
Would it make sense to hook this into all non-`GET` requests, rather than do it on a method-by-method basis?

Reply:
Not sure, `*/services/oauth2/token` seems to be a post.

Reply:
I reckon it might be worth just having an override for those cases - non-`GET` is generally correct.